troubleshooting/tools/tracevol.py

```diff
@@ -461,16 +466,19 @@ def get_subvol_group(arg):
     except ValueError as err:
         print(err, stdout)
         sys.exit()
-    cm_data = config_map['data']['config.json']
     subvol_group = ""
```

Comment:
default subvolumegroup will be csi isn't it?

Reply:
yeah, if this returns empty, then the caller check_subvol_in_cluster() passes csi

Reply:
instead of  returning empty and caller acting based  on the return value, can we define "csi"

Reply:
fixed